provisioning/baremetal_config.go

```diff
@@ -46,6 +46,7 @@ const (
 	bootIsoSource                  = "IRONIC_BOOT_ISO_SOURCE"
 	useUnixSocket                  = "unix"
 	useProvisioningDNS             = "provisioning"
+	customConfigPath               = "CUSTOM_CONFIG_DIR" // TODO(hroy): do we need this
```

Comment:
I haven't used this var, but I'm just wondering if we need this and some other variables such as `ironicDataPath` as inputs.